#! /bin/sh -e

# DP: disable large file support for GNU/Hurd

dir=.
if [ $# -eq 3 -a "$2" = '-d' ]; then
    pdir="-d $3"
    dir="$3/"
elif [ $# -ne 1 ]; then
    echo >&2 "usage: `basename $0`: -patch|-unpatch [-d <srcdir>]"
    exit 1
fi
case "$1" in
    -patch)
        patch $pdir -f --no-backup-if-mismatch -p1 < $0
        ;;
    -unpatch)
        patch $pdir -f --no-backup-if-mismatch -R -p1 < $0
        ;;
    *)
	echo >&2 "usage: `basename $0`: -patch|-unpatch [-d <srcdir>]"
        exit 1
esac
exit 0

diff -ru python2.3-2.3.2.orig/configure.in python2.3-2.3.2/configure.in
--- python2.3-2.3.2.orig/configure.in	2003-09-25 17:21:00.000000000 +0200
+++ python2.3-2.3.2/configure.in	2003-10-05 09:38:36.000000000 +0200
@@ -970,6 +970,9 @@
   use_lfs=no
 fi
 
+# Don't use largefile support anyway.
+use_lfs=no
+
 if test "$use_lfs" = "yes"; then
 # Two defines needed to enable largefile support on various platforms
 # These may affect some typedefs
